pkg/processor/build/builder.go

```diff
@@ -137,6 +141,9 @@ func NewBuilder(parentLogger logger.Logger, platform platform.Platform, s3Client
 		platform:    platform,
 		s3Client:    s3Client,
 		versionInfo: version.Get(),
+
+		// TODO: Remove in 1.16.x once we remove the deprecated `github` entry type
+		githubRegex: regexp.MustCompile(GithubURLRegexPattern),
```

Comment:
Doesn't Nuclio create a builder entity every time we do build?

Reply:
Do you suggest initializing it in some global variable?

Reply:
@TomerShor yep, otherwise it doens't make much sense to move it from the function (as it was done before), because we anyway compile regexp on each build